test/graphql/mutations/delete_profile_mutation_test.rb

```diff
@@ -79,4 +80,29 @@ class DeleteProfileMutationTest < ActiveSupport::TestCase
     assert_audited 'Removed profile'
     assert_audited @profile.policy.id
   end
+
+  context 'unauthorized user' do
+    setup do
+      stub_rbac_permissions(Rbac::COMPLIANCE_VIEWER, Rbac::INVENTORY_VIEWER)
+    end
+
+    should 'have the delete action rejected' do
+      second = FactoryBot.create(
+        :profile,
+        account: @user.account,
+        policy: @profile.policy,
+        external: true
+      )
+
+      assert_raises(GraphQL::UnauthorizedError, 'User is not authorized to access this action.') do
```

Comment:
If you're not catching the exception here but asserting the error from the HTTP request, you'll have coverage for the missing line.